src/datasets/other/addons_daily/reference.md

```diff
@@ -3,17 +3,122 @@
 # `addons_daily` Derived Dataset
 Contributers: Sarah Melancon, Ben Miroglio, Brian Wright, Daniel Thorn
 
-This ETL code produces daily aggregates of Firefox extensions. It supports the broader "Extention Data for Developers" Project.
+This ETL code produces daily aggregates of Firefox extensions. It supports the broader "Extension Data for Developers" Project.
```

Comment:
This is good for the repo README, but the context is a bit lost in the DTMO docs. Let's remove this sentence and focus on the dataset's details for DTMO.